src/v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -220,9 +226,62 @@ const processEvent = async (metadata, message, destination) => {
     }
 
     Object.values(createdPayload).forEach((data) => {
-      response.push(responseBuilder(metadata, data, destination));
+      responses.push(responseBuilder(metadata, data, destination));
     });
-    return response;
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const { body, method, params, endpoint, headers } of responses) {
+      // const { headers } = request;
+      const { customerId, listId } = params;
+
+      // step1: offlineUserDataJobs creation
+
+      const firstResponse = await createJob(endpoint, customerId, listId, headers, method);
+      if (
+        !firstResponse.success &&
+        !isHttpStatusSuccess(firstResponse?.response?.response?.status)
+      ) {
+        const processedResponse = processAxiosResponse(firstResponse);
+        const authErrorCategory = getAuthErrCategory(
+          processedResponse.status,
+          processedResponse.response,
+        );
+        throw new InvalidAuthTokenError(
+          `${processedResponse?.response?.error?.message} during offlineUserDataJobs creation`,
+          firstResponse?.response?.response?.status,
+          firstResponse,
+          authErrorCategory,
+        );
+      }
+
+      // step2: putting users into the job
+      let jobId;
+      if (firstResponse?.response?.data?.resourceName) {
+        // eslint-disable-next-line prefer-destructuring
+        jobId = firstResponse.response.data.resourceName.split('/')[3];
+      }
+      const secondResponse = await addUserToJob(endpoint, headers, method, jobId, body);
+      if (
+        !secondResponse.success &&
+        !isHttpStatusSuccess(secondResponse?.response?.response?.status)
+      ) {
+        const processedResponse = processAxiosResponse(firstResponse);
```

Comment:
This should be `secondResponse` right ? Am I missing any logic here ?

Reply:
Yes correct updated the code.